src/Symfony/Component/Process/PhpUnixExecutableFinder.php

```diff
@@ -0,0 +1,106 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Process;
+
+use Symfony\Component\Process\Contracts\CommandExecutorInterface;
+use Symfony\Component\Process\Contracts\ExecutableFinderInterface;
+use Symfony\Component\Process\Exception\PhpUnixExecutableInvalidVersionException;
+use Symfony\Component\Process\Exception\PhpUnixExecutableNotFoundException;
+
+/**
+ * An executable finder specifically designed for the PHP executable on Unix-based systems.
+ * This class allows searching for a specific version of PHP if provided.
+ *
+ * @author Pululu Kinaga Andre <pululuandre@gmail.com>
+ */
+class PhpUnixExecutableFinder
```

Comment:
Why making this Unix only instead of making it use the ExecutableFinder that works on Windows as well ?

Reply:
Hi @stof 
Thanks for your feedback, I just simplify my implementation